templates/rems.yaml

```diff
@@ -19,16 +19,16 @@ spec:
       initContainers:
       {{- if .Values.dbConnection.wait }}
       - name: database-wait
-        image: jbergknoff/postgresql-client
+        image: {{ include "imageRegistry" . }}jbergknoff/postgresql-client
```

Comment:
shall we use the official postgresql image or https://hub.docker.com/r/bitnami/postgresql or we can keep this as well, thinking we can make use of one image instead of using multiple?